usefulUtilities/scaleAvatar/scaleAvatar.js

```diff
@@ -0,0 +1,135 @@
+//
+//  scaleAvatar.js
+//
+//  Run script on client. In VR, when you hold down both grips and both triggers, 
```

Comment:
no need to change anything for this time! this is information that is duplicated between this file and the README. i personally don't think it's necessary to include info like this in both places; i only include it in the readme.

for future projects (or for this one, if you like), you don't have to include this twice.